src/main/java/inf112/isolasjonsteamet/roborally/gui/MapRendererWidget.java

```diff
@@ -31,6 +30,9 @@ public MapRendererWidget(ClientBoard board, int tileSize) {
 		sizeChanged();
 	}
 
+	/**
```

Comment:
`{@inheritDoc}` in here. Even more important for some of the other methods in here, as the JavaDoc specified here is just plain wrong in some cases.